frontend/src/components/SettingsPanel.tsx

```diff
@@ -277,26 +278,6 @@ export function SettingsPanel({
                         </Tooltip>
                       </TooltipProvider>
                     )}
-                    {currentPipeline.modified && (
```

Comment:
I don't think the `modified` attribute was adding any value so opted to just remove to simplify since we were moving a bunch of attributes to the backend.